<launch>
  <node name="zone_detection" pkg="endorse_safety" type="cluster_zone_detection" >
	<remap from="/vector_prediction" to="/pointcloud_lidar_processing/detected_objects"/>
  </node>
  <!--node name="cluster_detection" pkg="endorse_safety" type="endorse_cluster_search" >
	<remap from="/rslidar_points" to="/velodyne_points"/>
  </node-->
  <!--include file="$(find rslidar_pointcloud)/launch/rs_lidar_16.launch" />
   <node pkg="tf" type="static_transform_publisher" name="base2rslidar" args="1.1 0. -.75 1.57 0 0 'rslidar' 'base_link' 100"/-->
</launch>
